### DISCUSSION TITLE
unistyles support for  react-native-tvos

### DISCUSSION THREAD

**Discussion by jariwalabhavesh** [jpudysz/react-native-unistyles#171] [Feature Requests]
Hello All,

I hope all folks are doing good.

I am using unistyles with my react-native project and it is working like charm. I also development with TV os development using [react-native-tvos](https://github.com/react-native-tvos/react-native-tvos).

As unistyles is working for react-native and react-native-tvos is also based on react-native. So I decided to try out unistyles with react-native-tvos.

I am successfully able to integrate unistyles in react-native-tvos for android-tv. I had tried colors config for theme and they are working fine.

But when I am trying on react-native-tvos for apple-tv platform it is giving error regarding unsupported platform while pod installation. 
<img width="575" alt="Screenshot 2024-03-31 at 4 21 12 PM" src="https://github.com/jpudysz/react-native-unistyles/assets/10993666/75335f18-3ea3-47c6-9935-fc94dda1f01a">

It will be awesome if we have unistyles support for react-native-tvos 's apple-tv platform.

Basically using react-native for mobile app & for OTT platform (Android-TV & Apple-TV) (using react-native-tvos) will provide us great convenience to reuse api calls, styling etc.. By supporting in OTT platform we can take advantage of breakpoints & mediaquery feature in single codebase. Also reusing component will be easy.

Currently my component use unistyle in react-native-mobile app but for react-native-tvos platform I need to modify style to tradition react-native stylesheet.

Let me know what other folks think on this

**Comment by jpudysz** [comment]
Hello! 

That's super cool, I'm amazed that for Android TV it works out of the box.

To answer your question please take a look here: https://github.com/jpudysz/react-native-unistyles/discussions/75

Support for tvOS will be available in the next major version (2.6.0) ☺️

**Comment by jariwalabhavesh** [comment]
@jpudysz 

Hey That's good news that it's coming soon in v2.6.0. Excited to review that.

Thanks for update

**Reply by jpudysz** [reply]
It's already available in 2.6.0-rc.1

You can install it with:

```shell
yarn add react-native-unistyles@next
```

Should be stable, but I need more tests, and build some UI :)
